includes/payment-methods/class-wc-stripe-upe-payment-method-blik.php

```
@@ -87,20 +87,24 @@ public function get_retrievable_type() {
 	 * @param bool $show_optimized_checkout_instruction Deprecated. Whether to show optimized checkout instructions.
 	 * @return string
 	 */
-	public function get_testing_instructions( $show_optimized_checkout_instruction = false ) {
+	public function get_testing_instructions( $show_optimized_checkout_instruction = false, bool $include_test_mode_label = true ) {
```

PHPMD (2.15.0)

[error] The method get_testing_instructions has a boolean flag argument $show_optimized_checkout_instruction, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)